config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml

```
@@ -42,11 +58,13 @@ spec:
       kind: CertManager
       name: certmanagers.operator.openshift.io
       version: v1alpha1
-  description: cert-manager-operator
-  displayName: cert-manager-operator
+  description: |
+    The cert-manager Operator for Red Hat OpenShift provides seamless support for cert-manager, which automates certificate management.
+    For more information, see [cert-manager.io](https://cert-manager.io).  displayName: cert-manager-operator
```

```suggestion
    For more information, see [cert-manager.io](https://cert-manager.io).
```
Forgot removing trailing displayName: cert-manager-operator?

---

/hold
Adding /hold. Feel free to unhold if above comment is not right.